packages/instantsearch.js/src/middlewares/createInsightsMiddleware.ts

```diff
@@ -225,11 +225,27 @@ export function createInsightsMiddleware<
           immediate: true,
         });
 
+        // @ts-ignore
+        const insightsClientWithLocalCredentials = (method, payload) => {
```

Comment:
Not yet, we need to release the new version of Insights so that the types are up to date.

I can do it in a different PR if we don't release it before we're ready to merge this one.